iot/iot-tenant-service/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM enmasseproject/java-base:8-11
+
+ARG version
+ARG commit
+ENV VERSION=${version} COMMIT=${commit}
```

Comment:
The maven version should be used, otherwise the release tag artifact will not be named correctly.

See address-space-controller/Dockerfile for instance.

Reply:
Thx! I wasn't aware of that. Should be fixed now.